.github/workflows/update-tree-sitter-wasm-file.yml

```diff
@@ -44,26 +46,26 @@ jobs:
       run: npm install
 
     - name: Update tree-sitter wasm
-      run: bash scripts/update-tree-sitter-wasm.sh
+      run: bash scripts/update-${{ inputs.tree-sitter-name }}-wasm.sh
 
     - name: Verify file changes
       uses: tj-actions/verify-changed-files@v17
       id: verify-changed-files
       with:
         # The script generates a new wasm file and replaces the existing one. Git will treat it as a different file even it is generated with the same commit and CLI  
-        # Hence, we only compare the parser.info. It should be enough to tell the difference
+        # Hence, we only compare the .info file. It should be enough to tell the difference
         files: |
-            server/parser.info
+            server/${{ inputs.tree-sitter-name }}.info
 
     - name: Create pull request
       if: steps.verify-changed-files.outputs.files_changed == 'true'
       uses: peter-evans/create-pull-request@v5
       with:
         add-paths: |
-          server/parser.info
-          server/tree-sitter-bitbake.wasm
-        title: Auto update tree-sitter-bitbake wasm file
-        commit-message: Auto update tree-sitter-bitbake wasm file and parser info
-        branch: update-tree-sitter-wasm-file
+          server/${{ inputs.tree-sitter-name }}.info
+          server/${{ inputs.tree-sitter-name }}.wasm
+        title: Auto update ${{ inputs.tree-sitter-name }} wasm file
+        commit-message: Auto update ${{ inputs.tree-sitter-name }} wasm file and parser info
```

Comment:
Have you been able to launch a test run through github actions?

Reply:
We can compare with this successful run: https://github.com/yoctoproject/vscode-bitbake/actions/runs/9225032932/job/25381628666
I relaunched it and it also failed with my user, so its not a user issue. I get no issue if I push by hand it's a service user issue.
This guide https://docs.github.com/en/actions/security-guides/automatic-token-authentication#modifying-the-permissions-for-the-github_token
mentions we could need to add permissions specific to the new workflows.
```
You can see the permissions that GITHUB_TOKEN had for a specific job in the "Set up job" section of the workflow run log. For more information, see "[Using workflow run logs](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/using-workflow-run-logs)."
```
That is the error! 
![image](https://github.com/yoctoproject/vscode-bitbake/assets/19970889/d7366f1e-b867-4198-9b4d-f3eb0a86a9a3)
So we need to change the permissions for the new workflows like @WilsonZiweiWang has probably already done for the update-ref workflow

Reply:
I bet the problem would indeed solve itself once merged because right now it runs as an external PR.
Let's merge and check, we can fix it if it's not working still.